src/element.ts

```diff
@@ -616,6 +616,29 @@ export class Element {
     return this.#textMetrics;
   }
 
+  /** Measure the text using the FontInfo related with key. */
+  static getElementWidth(text: string, key = ''): number {
+    let txtCanvas: HTMLCanvasElement | OffscreenCanvas | undefined = Element.#txtCanvas;
+    // Create the canvas element that will be used to measure text.
+    if (!txtCanvas) {
+      if (typeof document !== 'undefined') {
+        txtCanvas = document.createElement('canvas'); // Defaults to 300 x 150. See: https://www.w3.org/TR/2012/WD-html5-author-20120329/the-canvas-element.html#the-canvas-element
+      } else if (typeof OffscreenCanvas !== 'undefined') {
+        txtCanvas = new OffscreenCanvas(300, 150);
+      }
+      Element.#txtCanvas = txtCanvas;
+    }
```

Comment:
Is this duplicated from Element.measureText()?

If so, we should extract this into a separate method that initializes the Element.#txtCanvas if it does not exist yet.

Maybe `Element.getTextMeasurementCanvas()` and put it right after Element.setTextMeasurementCanvas()? If it is undefined, it creates a new one. If it is not undefined, it just returns the current one.

Reply:
done